email_functions.py

```diff
@@ -1,43 +1,33 @@
-'''
-reading_mail() - takes no input and returns a dictonary with mail argumnets as dict['subject']  , dict['msg_body']
-send_mail() - takes 2 arguments - subject , message and returns 1 if message is sent successfully and 0 is a error occurs
-'''
-import gmail
-import smtplib
-import json
-from email.mime.text import MIMEText
+# email_functions.py
+
 import Logger
 import os
+import requests
 
 logger = Logger.Logger(name='RunLog')
-f_email = os.getenv('SENDER_EMAIL_ID')
-t_email = os.getenv('SENDEE_EMAIL_ID')
-passwd = os.getenv('EMAIL_PASS')
+key = os.getenv('MAILGUN_API_KEY')
+sender = os.getenv('SENDER_EMAIL_ID')
+recipient = os.getenv('RECIPIENT_EMAIL_ID')
+sandbox = os.getenv('MAILGUN_SANDBOX_URL')
 
 
-def reading_mail () :  # this function returns a dictionary with email arguments
-	g = gmail.Gmail()
-	g.login(f_email, passwd)  #logging in to gmail server
-	unread = g.inbox().mail(unread=True)  #getting all unread mails. It returns all the blank mials
-	unread[-1].fetch()   # getting all the mail attributes like body,subject etc
-	mail_args = {'subject' : unread[-1].subject , 'msg_body' : unread[-1].body}
-	unread[-1].read()  #marking the mail as read
-	return mail_args
-	g.logout()  #logging out
+def send_mail(mail_subject, mail_body) :
+    #sending mail
+    request_url = 'https://api.mailgun.net/v2/{0}/messages'.format(sandbox)
 
+    request = requests.post(request_url, auth=('api', key), data={
+        'from': sender,
+        'to': recipient,
+        'subject': mail_subject,
+        'text': mail_body
+    })
+
+    print('Status: {0}'.format(request.status_code))
+    print('Body:   {0}'.format(request.text))
+
+    if (request.status_code==200):
```

Comment:
Just return the status code here. There can be more changes later. If the request ends in an error, then return the error.